### PR TITLE
Add description for FixedStatusCode

### PR DESCRIPTION
**Pull request by mariolyon**
This PR addresses issue: https://github.com/softwaremill/tapir/issues/155 and allows the assigning of a description to a response status code when using FixedStatusCode.

### CODE REVIEW

File: docs/openapi-docs/src/main/scala/tapir/docs/openapi/EndpointToOperationResponse.scala [hunk 1]
```diff
@@ -65,13 +65,18 @@ private[openapi] class EndpointToOperationResponse(objectSchemas: ObjectSchemas,
     }
     val body = bodies.headOption
 
-    val description = body.flatMap(_._1).getOrElse("")
+    val descriptions = outputs.collect {
+      case EndpointOutput.FixedStatusCode(sc, info) if info.description.isDefined => info.description.get
```
Comment (adamw) [inline docs/openapi-docs/src/main/scala/tapir/docs/openapi/EndpointToOperationResponse.scala]: maybe nicer: `case FixedStatusCode(sc, EndpointInfo(_, _, Some(description), _)))`

File: core/src/main/scala/tapir/internal/server/EncodeOutputs.scala [hunk 2]
```diff
@@ -59,7 +60,7 @@ class EncodeOutputs[B](encodeOutputBody: EncodeOutputBody[B]) {
   }
 }
 
-case class OutputValues[B](body: Option[B], headers: Vector[(String, String)], statusCode: Option[StatusCode]) {
+case class OutputValues[B](body: Option[B], headers: Vector[(String, String)], statusCode: Option[StatusCode], info: Option[Info[Unit]]) {
```
Comment (adamw) [inline core/src/main/scala/tapir/internal/server/EncodeOutputs.scala]: do you need the `info` here?

**Comment by adamw** [comment]
Thanks! Just two minor comments :)

**Comment by mariolyon** [comment]
ok, thanks will revise...

**Pull request merged**